src/client.rs

```diff
@@ -1529,6 +1529,25 @@ pub async fn handle_test_delay(t: TestDelay, peer: &mut Stream) {
     }
 }
 
+#[inline]
+#[cfg(target_os = "macos")]
+fn check_scroll_on_mac(mask: i32, x: i32, y: i32) -> bool {
```

Comment:
I am not sure if this function will work in flutter version, Here is some relevant code.

https://github.com/rustdesk/rustdesk/blob/1b7950fc428e0c0e3f68d57efc66837c0a05d256/src/flutter_ffi.rs#L828

https://github.com/rustdesk/rustdesk/blob/1b7950fc428e0c0e3f68d57efc66837c0a05d256/flutter/lib/models/input_model.dart#L287

Reply:
Have you tested in flutter version? @xxrlzzz

Reply:
I had test on iOS, download from App Store

Reply:
You say flutter version, do your mean flutter for mac version? I only found flutter build for mobel.

Reply:
Yes, flutter for mac version. Windows, Linux, and Mac currently have flutter versions, They will all call the `check_scroll_on_mac` function

Reply:
I don't have a track pad to test if this function works properly in flutter

Reply:
patch updated
flutter build keep crash with log, seems not my problem
```
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: MissingPluginException(No implementation found for method getAll on channel plugins.flutter.io/shared_preferences_macos)
```

Reply:
@fufesou removed share_pref dep in latest commit, you can remove it from  pub config, and cleanup cache and pod cache etc.